app/policies/public_page_policy.rb

```diff
@@ -14,7 +14,7 @@ def template_index?
   end
 
   def template_export?
-    @object.is_default || @object.org.funder?
+    @object.present? && ( @object.is_default || @object.org.funder? ) && @object.published
```

Comment:
Not necessary but you should be able to add a `?` on `is_default?` and `published?`

Reply:
Good call, I'll update those.